ncm-ceph/src/main/pan/components/ceph/schema.pan

```diff
@@ -208,23 +208,26 @@ type ceph_mds = {
 
 @documentation{ ceph cluster-wide config parameters }
 type ceph_cluster_config = {
-    'fsid'                      : string # Should be generated with uuidgen
-    'filestore_xattr_use_omap'  : boolean = true
-    'osd_journal_size'          : long(0..) = 10240
-    'mon_initial_members'       : string [1..]
-    'public_network'            : type_network_name
-    'cluster_network'           ? type_network_name
-    'auth_service_required'     : string = 'cephx' with match(SELF, '^(cephx|none)$')
     'auth_client_required'      : string = 'cephx' with match(SELF, '^(cephx|none)$')
     'auth_cluster_required'     : string = 'cephx' with match(SELF, '^(cephx|none)$')
-    'osd_pool_default_pg_num'   : long(0..) = 600
-    'osd_pool_default_pgp_num'  : long(0..) = 600
-    'osd_pool_default_size'     : long(0..) = 3
-    'osd_pool_default_min_size' : long(0..) = 2
+    'auth_service_required'     : string = 'cephx' with match(SELF, '^(cephx|none)$')
```

Comment:
indeed :smile: 
I would rather keep it this way, otherwise all clusters not specifying this explicitly will be unauthenticated..